front-end/src/components/Students/Table/Table.module.scss

```diff
@@ -0,0 +1,77 @@
+@import 'styles/colors.scss';
+@import 'styles/font.scss';
+
+table {
```

Comment:
Are you sure that these styles only for your table not for all tables in app?

Reply:
this style will be changed, after Sasha's style update on this table